app/controllers/api/v1/graphql_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class GraphqlController < ActionController::API
+      # If accessing from outside this domain, nullify the session
+      # This allows for outside API access while preventing CSRF attacks,
+      # but you'll have to authenticate your user separately
+      # protect_from_forgery with: :null_session
+
+      def execute
+        render json: result
+      rescue StandardError => e
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/bde03b9355743c3ad209800a745b8cc6547ad670/app/controllers/api/v1/graphql_controller.rb#L13"></span>Has the variable name ‘e’</td>
    </tr>
  </tbody>
</table>